src/annotations/components/annotationForm/AnnotationTimeInput.tsx

```diff
@@ -77,20 +79,11 @@ export const AnnotationTimeInput: FC<Props> = (props: Props) => {
 
     if (isValidTimeFormat(event.target.value)) {
       if (timeZone === 'UTC') {
-        props.onChange(
-          moment
-            .utc(event.target.value, timeFormat)
-            .toDate()
-            .toISOString()
-        )
+        props.onChange(setTimeToUTC(event.target.value))
```

Comment:
why not do the same thing you're doing below?
```suggestion
        props.onChange(new Date(event.target.value).toISOString())
```

Reply:
We'd need a UTC value, for the time value passed. Hence. If that makes sense.

Reply:
K, is it a selector or a utility function that exists within a selector file?

Reply:
Yah! its in `src/dashboards/selectors`

Reply:
it seems like a utility function. Needed in the selectors.

Reply:
Gotcha.